pkg/annotations/dynamic-network-status.go

```diff
@@ -0,0 +1,79 @@
+package annotations
+
+import (
+	"encoding/json"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+
+	nettypes "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+	multusapi "gopkg.in/k8snetworkplumbingwg/multus-cni.v3/pkg/server/api"
+)
+
+func AddDynamicIfaceToStatus(currentPod *corev1.Pod, networkSelectionElement *nettypes.NetworkSelectionElement, response *multusapi.Response) (string, error) {
+	currentIfaceStatus, err := podDynamicNetworkStatus(currentPod)
+	if err != nil {
+		return "", err
+	}
+
+	newIfaceStatus := nettypes.NetworkStatus{
+		Name: fmt.Sprintf("%s/%s", networkSelectionElement.Namespace, networkSelectionElement.Name),
+	}
+
+	if response != nil && response.Result != nil {
+		for _, iface := range response.Result.Interfaces {
+			newIfaceStatus.Interface = iface.Name
+			newIfaceStatus.Mac = iface.Mac
+		}
```

Comment:
Don't we have to add multiple interfaces from the response instead of the last one ?